js/prenotazioni.js

```diff
@@ -0,0 +1,146 @@
+fetch ("js/libri.json")
+    .then( response => response.json() )
+    .then( data => {
+
+    let array = data;
+    console.log(data);
+    for( let i = 0; i < data.length; i++ ){
+        let row = document.createElement("tr");
+        row.id = "riga" + (i + 1);
+
+        immagini(i, data , row);
+        stato( i,data, row );
+        titolo( i, data, row );
+        anno( i, data, row );
+        azione( data[i].stato, row, i);
```

Comment:
Questi metodi vanno benissimo, ma li nominerei in modo da rendere più evidente a cosa servono. Es. `renderizzaImmagini`.